lib/params/MapperFunction.cpp

```diff
@@ -133,6 +137,8 @@ MapperFunction::MapperFunction(
 
 	m_opacityMaps = new ParamsContainer(*(rhs.m_opacityMaps));
 	m_opacityMaps->SetParent(this);
+
+	_dirtyBit = true;
 }
 
 MapperFunction &MapperFunction::operator=( const MapperFunction& rhs ) {
```

Comment:
What's the rational for this assign constructor to always construct _dirtyBit = true, instead of get the value from rhs?

Reply:
Fair point.